weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -2845,18 +2845,20 @@ def _setup_completion_model_info(
     elif model_name.startswith("coreweave/"):
         # See https://docs.litellm.ai/docs/providers/openai_compatible
         # but ignore the bit about omitting the /v1 because it is actually necessary
-        req.inputs.model = "openai/" + model_name.replace("coreweave/", "", 1)
+        req.inputs.model = model_name.replace("coreweave/", "openai/", 1)
+
         provider = "custom"
-        base_url = "https://api.inference.wandb.ai/v1"
+
+        base_url = wf_env.inference_service_base_url()
```

Comment:
What will happen if base_url is None?

Reply:
It will fail. Let me default to prod.

Reply:
Added a default value set to the prod URL.